src/themes/light.scss

```diff
@@ -8,6 +8,13 @@
   --wxc-icon--active-color: #{$wxc-green-50};
   --wxc-icon--muted-color: #{$wxc-red-60};
 
+  --wxc-badge--spinner-border-color: #D1D3D4;
```

Comment:
is this needed? it shouldn't be. If it's really needed, it must not belong to the Badge (it does not know that it contains a color), instead, it must be styled in the component that uses the Badge, and please use a color code.